tests/e2e/specs/edit-story/example.js

```diff
@@ -25,10 +25,19 @@ import { createNewStory, exportData, popStats } from '../../utils';
 
 describe('Selection with new element', () => {
   let safezoneBox;
+  let errors;
 
   beforeEach(async () => {
     await createNewStory();
 
+    errors = [];
+    page.on('error', (error) => {
+      errors.push({cat: 'error', error});
+    });
+    page.on('pageerror', (error) => {
+      errors.push({cat: 'pageerror', error});
```

Comment:
I thought the `observeConsoleLogging` util in `bootstrap.js` would already fail tests if any errors are logged. 

If this is needed, adding it to `bootstrap.js` to run for all tests would make sense

Reply:
Sure. I might have missed it in some trials. I'll try again.